### PR TITLE
[Example] Support resnet50

### LINKED ISSUE #351: [Example] OnnxModelImporter fails to load resnet50_8.onnx of onnx.js
onnx.js: https://github.com/Microsoft/onnxjs
resnet50_8.onnx used by onnx.js example: https://github.com/Microsoft/onnxjs-demo/tree/data/data/benchmark/model-onnx
ONNX ResNet model: https://github.com/onnx/models/tree/master/resnet50

OnxxModelImporter throws `Invalid pads` error.

### PR DESCRIPTION
**Pull request by pinzhenx**
Fix #351

- add default value for the case of 'no padding'
- add Sum op for onnx
- update preprocess param

### CODE REVIEW

File: examples/util/onnx/OnnxModelImporter.js [hunk 1]
```diff
@@ -451,51 +451,89 @@ class OnnxModelImporter {
...
-          let output = node.output[0];
-          if (nextNode && nextNode.opType === 'Relu' && node.output[0] === nextNode.input[0]) {
-            // Fuse relu
-            inputs.push(this._addScalarInt32(this._nn.FUSED_RELU));
-            i++;
-            console.log(`  fuse relu: ${nextNode.output[0]} -> ${output}`);
-            output = nextNode.output[0];
-          } else {
-            inputs.push(this._addScalarInt32(this._nn.FUSED_NONE));
+          if (node.opType === 'Sum' && node.input.length === 1) {
```
Comment (huningxin) [inline examples/util/onnx/OnnxModelImporter.js]: do we need to skip `Sum` with number of inputs > 2?

Comment (pinzhenx) [reply examples/util/onnx/OnnxModelImporter.js]: I implement this op previously when enabling [DUC segmentation](https://github.com/onnx/models/tree/master/models/semantic_segmentation/DUC). It has used multiple-input Sum. So I keep these lines for future use.

Comment (huningxin) [reply examples/util/onnx/OnnxModelImporter.js]: However, current WebNN POC API doesn't support `Sum`. I understand here we have to map the `Sum` with two inputs to `Add`. Is my understanding correct?

Comment (pinzhenx) [reply examples/util/onnx/OnnxModelImporter.js]: Yes. And I understand that I should hold it for now until we support it natively. I will create a new commit later.

File: examples/util/onnx/OnnxModelImporter.js [hunk 2]
```diff
@@ -703,6 +703,55 @@ class OnnxModelImporter {
           console.log(`  output ${output}: [${outputDims}]`);
           opCode = this._nn.RESHAPE;
         } break;
+        case 'Sum': {
```
[the remark this answers is not in the document]
Comment (pinzhenx) [reply examples/util/onnx/OnnxModelImporter.js]: sorry for forgetting to take it off. it's been updated.

**Pull request merged**